test/integration/full/smoke.spec.js

```diff
@@ -0,0 +1,28 @@
+import { describe, it, expect } from '@jest/globals';
+import { configureServer } from '../../../src/server';
+import supertest from 'supertest';
+import * as cheerio from 'cheerio';
+
+describe('Smoke test', () => {
+  let server;
+  let request;
+
+  beforeEach(async () => {
+    server = await configureServer();
+    await server.start();
```

Comment:
I thought the point of Cheerio was that you didn't need to start the server. Is this incorrect?

Reply:
For smoke tests we do want to start the server as we want everything up (i.e. networking etc). Cheerio is used in both full and narrow tests regardless of the server start up approach as a way to query on page components.